cEpiabm/test/utility/test_distance_metrics.cpp

```diff
@@ -0,0 +1,50 @@
+#include "utility/distance_metrics.hpp"
+
+#include "../catch/catch.hpp"
+
+using namespace epiabm;
+
+TEST_CASE("utility/distance_metrics: test distance metric init", "[DistanceMetrics]")
+{   // tests the Dist function returns the same as the DistEuclid
+    DistanceMetrics subject = DistanceMetrics();
+}
+
+TEST_CASE("utility/distance_metrics: test distance metric", "[DistanceMetrics]")
+{   // tests the Dist function returns the same as the DistEuclid
+    auto loc = std::make_pair(1.0, 1.0);
+    double subject = DistanceMetrics::Dist(loc, loc);
+    double euclid = DistanceMetrics::DistEuclid(loc, loc);
+    REQUIRE(subject == euclid);
+
+    auto loc2 = std::make_pair(1.0, 0.0);
```

Comment:
```suggestion
    auto loc2 = std::make_pair(-1.0, 2.0);
```
Tiny thing but if we have two distances it might be good for them to have x and y separation to check this is initialised properly (and crossing the origin for the same reason). I know we aren't testing to the same level on the C++ but thought I'd suggest this as it seems an easy change to make! Ditto on the test cases below